src/command/remark.ts

```diff
@@ -354,3 +365,13 @@ export type Remark =
   | IWindPeakCommandRemark
   | IWindShiftRemark
   | IWindShiftFropaRemark;
+
+export type RemarkDated =
+  | RemarkBase
+  // Canadian commands below
+  | INextForecastByRemarkDated;
+
+export type Remark =
+  | RemarkBase
+  // Canadian commands below
+  | INextForecastByRemark;
```

Comment:
These also need to be exported in index.ts:

INextForecastByRemarkDated
INextForecastByRemark
RemarkDated

(no need to export RemarkBase, it's internal)

Reply:
done